requirements.txt

```diff
@@ -4,7 +4,7 @@ bottleneck==1.4.2
 numexpr==2.10.1
 pandas-ta==0.3.14b
 
-ccxt==4.4.20
```

Comment:
that's a pointless change and will cause a conflict tomorrow - and from there on every week the PR is open.

best leave version bumps to dependabot - unless there's a specific "Min version" requirement - where still - this would be the wrong place to bump.

Reply:
sorry, new to all this. reverted